### PR TITLE
Feature update

### PR DESCRIPTION
**Pull request by Kajochi**

### CODE REVIEW

File: backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java [hunk 1]
```diff
@@ -39,26 +39,55 @@ void WhenListIsEmptyReturnEmptyList() throws Exception {
...
-                                "    \"amount\": 34,\n" +
-                                "    \"category\": \"INCOME\",\n" +
-                                "    \"interval\": \"MONTHLY\"\n" +
-                                "}")
-        )
-        //Then
+                        MockMvcRequestBuilders.post("/api/entries")
+                                .contentType(MediaType.APPLICATION_JSON)
+                                .content("{\n" +
+                                        "    \"title\": \"testTitle\",\n" +
```
Comment (StormTrooper-007) [inline backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java]: `
@Autowired
Objectmapper objectMapper;

String requestBody = objectMapper.writeValueAsString(your-Class-Instance-variable);

 .content(requestBody);
`

**Review by StormTrooper-007 — APPROVED** [review]
Objectmapper will help you convert your object to json instead of writing this - 
"{\n" +
                                "    \"title\": \"testTitle\",\n" +
                                "    \"description\": \"testDescription\",\n" +
                                "    \"date\": \"2023-12-03\",\n" +
                                "    \"amount\": 34,\n" +
                                "    \"category\": \"INCOME\",\n" +
                                "    \"interval\": \"MONTHLY\"\n" +
                                "}")

you can try it  ;)

**Review by tasmslm90 — APPROVED** [review]
well done

File: backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java (hunk 1 above)
Comment (Kajochi) [reply backend/src/test/java/de/neuefische/capstone/backend/entries/EntriesIntegrationTest.java]: Thank you for this suggestion. This is much better;)

**Pull request merged**